docs/known_issues.rst

```diff
@@ -203,6 +203,23 @@ Colored printing of log messages and other colored text does work in Windows
 but only when running in the IPython console.  Colors are not currently
 supported in the basic Python command-line interpreter on Windows.
 
+numpy.int64 does not decompose input
+------------------------------------
+::
+
+    >>> np.int64((15 * u.km) / (15 * u.imperial.foot))
+    1
+    >>> np.int((15 * u.km) / (15 * u.imperial.foot))
+    3280
+
+    >>> int is np.int
```

Comment:
I would remove this line and the example below in favor of an example above showing what happens with `int((15 * u.km) / (15 * u.imperial.foot))`.